RELEASE_NOTES.md

```diff
@@ -1,5 +1,152 @@
+- The `podman pod stop`, `podman pod pause`, `podman pod unpause`, and `podman pod kill` commands now work on multiple containers in parallel and should be significantly faster.
+- The `podman search` command now supports a `--list-tags` option to list all available tags for a single image in a single repository.
+- The `podman search` command can now output JSON using the `--format=json` option.
+- The `podman diff` and `podman mount` commands now work with all containers in the storage library, including those not created by Podman. This allows them to be used with Buildah and CRI-O containers.
+- The `podman container exists` command now features a `--external` flag to check if a container exists not just in Podman, but also in the storage library. This will allow Podman to identify Buildah and CRI-O containers.
+- The `--tls-verify` and `--authfile` flags have been enabled for use with remote Podman.
+- The `/etc/hosts` file now includes the container's name and hostname (both pointing to localhost) when the container is run with `--net=none` ([#8095](https://github.com/containers/podman/issues/8095)).
+- The `podman events` command now supports filtering events based on the labels of the container they occurred on using the `--filter label=key=value` option.
+- The `podman volume ls` command now supports filtering volumes based on their labels using the `--filter label=key=value` option.
+- The `--volume` and `--mount` flags to `podman run` and `podman create` now support two new mount propagation options, `unbindable` and `runbindable`.
+- The `name` filter for `podman pod ps` now matches based on a regular expression, instead of requiring an exact match.
+
+### Changes
+- The `podman network rm --force` command will now also remove pods that are using the network ([#7791](https://github.com/containers/podman/issues/7791)).
+- The `podman volume rm`, `podman network rm`, and `podman pod rm` commands now return exit code 1 if the object specified for removal does not exist, and exit code 2 if the object is in use and the `--force` flag was not given.
+- If `/dev/fuse` is passed into Podman containers as a device, Podman will open it before starting the container to ensure that the kernel module is loaded on the host and the device is usable in the container.
+- Global Podman options that were not supported with remote operation have been removed from `podman-remote` (e.g. `--cgroup-manager`, `--storage-driver`).
+- Many errors have been changed to remove repetition and be more clear as to what has gone wrong.
+- The `--storage` option to `podman rm` is now enabled by default, with slightly changed semantics. If the given container does not exist in Podman but does exist in the storage library, it will be removed even without the `--storage` flag. If the container exists in Podman it will be removed normally. The `--storage` flag is now deprecated and will be removed in a future release.
```

Comment:
```suggestion
- The `--storage` option to `podman rm` is now enabled by default, with slightly changed semantics. If the given container does not exist in Podman but does exist in the storage library, it will be removed even without the `--storage` option. If the container exists in Podman it will be removed normally. The `--storage` option is now deprecated and will be removed in a future release.
```